make.paws/R/sdk_helper.R

```diff
@@ -0,0 +1,93 @@
+#' @import data.table
+#' @importFrom yaml write_yaml
+#' @importFrom devtools release
+
+#' @title Check paws sdk
+#' @param in_dir Directory containing paws sdk packages.
+#' @param path Path to output paws sdk check results.
+#' @param keep_notes Include note
+#' @return A list of any warnings, errors and notes within paws sdk.
+#' @export
+paws_check <- function(in_dir = "../cran", path, keep_notes = FALSE){
+  checks <- list()
+
+  for (package in list.dirs(in_dir, recursive = FALSE)) {
+    checks[[basename(package)]] <- devtools::check(package, cran = TRUE)
```

Comment:
Maybe want to wrap this in something that suppresses output, like `sink(...); devtools::check(...); sink()` or something. I never remember what the right function is.